StaySphere.Api/StaySphere.Domain/Interfaces/Repositories/ICommonRepository.cs

```diff
@@ -11,7 +11,6 @@ public interface ICommonRepository
         public IPositionRepository Position { get; }
         public IReviewRepository Review { get; }
         public IRoomRepository Room { get; }
-        public int SaveChanges();
```

Comment:
Is it really necessary to delete the synchronous version of this method? Can't we have both so that the one who will use it can choose between two options?